concent_api/core/tests/test_integration_get_task_result.py

```diff
@@ -356,6 +356,7 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_failu
 
         # Test FileTransferToken message
         self.assertIsInstance(message_file_transfer_token, message.FileTransferToken)
+        self.assertIsNot(message_file_transfer_token.sig,  None)
```

Comment:
You should also check that `sig` is not equal to the output of `fake sign` (i.e. a byte string containing just zeros). Actually it would be best to verify the signature here. Otherwise you can't be sure it's correct.